web/modules/custom/rdf_entity/rdf_entity.module

```diff
@@ -2,6 +2,152 @@
 
 /**
  * @file
- * Contains Drupal\rdf_entity\rdf_entity.module
+ * Main functions and hook implementations of the RDF Entity module.
  */
 
+use \Drupal\Core\Form\FormStateInterface;
+use \Drupal\field_ui\Form\FieldStorageConfigEditForm;
+use \Drupal\field\Entity\FieldStorageConfig;
+use \Drupal\Core\Render\Element;
+use \Drupal\rdf_entity\RdfInterface;
+/**
+ * Implements hook_form_alter().
+ */
+function rdf_entity_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id != 'field_storage_config_edit_form') {
+    return;
+  }
```

Comment:
Use `hook_form_FORM_ID_alter()` instead of `hook_form_alter()` if you are just altering a single form.